src/View/layout.html.twig

```diff
@@ -5,6 +5,10 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="/assets/images/favicon.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/uikit@3.17.6/dist/css/uikit.min.css" />
+    <link rel="stylesheet" href="/assets/css/contact_form.css">
+    
+    <link rel="stylesheet" href="/assets/css/variables.css">
```

Comment:
Si tu utilises des variables dans les autres fichiers vaut mieux le mettre directement après Uikit pour que les variables se chargent avant tous les autres fichiers de style.

Reply:
c'est noté